DISCOVER/_health_protocols.md

```diff
@@ -0,0 +1,32 @@
+## Infectious Disease Precautions and Protocols
+Protocols
```

Comment:
Not sure what this line is doing but probably don't need it. Was it just a mistake?